dashboard/src/app/weatherProperties.ts

```diff
@@ -38,6 +38,14 @@ const weatherProperties = {
     icon: 'mdi-water-percent',
     title: 'Luchtvochtigheid op dit moment',
     unit: '%'
+  },
+  wbgt: {
+    property: 'wbgt',
+    name: 'Gevoelstemperatuur',
```

Comment:
@bmesuere after a discussion with @scaluwae we think it is best to name it _Gevoelstemperatuur (WBGT)_, because the WBGT is one of many proxies for the gevoelstemperatuur.